streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedCachingPersistentWindowStoreTest.java

```diff
@@ -138,51 +143,70 @@ public void setUp() {
 
     @After
     public void closeStore() {
-        cachingStore.close();
+        try {
+            cachingStore.close();
+        } catch (final RuntimeException runtimeException) {
```

Comment:
This may hide cases when the exception is genuine.

Alternatively, create a function:
```
private void verifyAndTearDownCloseTests() {
        verify(underlyingStore).close();
        verify(cache).flush(CACHE_NAMESPACE);
        verify(cache).close(CACHE_NAMESPACE);

        // resets the mocks created in #setUpCloseTests(). It is necessary to
        // ensure that @After works correctly.
        reset(cache);
        reset(underlyingStore);
    }
```

And call it at the end of each close test.

Reply:
Thanks for the suggestion. I have updated the PR